_docs/gui.rst

```diff
@@ -2,7 +2,96 @@ gui
 ======
 
 
+Design Requirements
+-------------------
+
+* Display Values
+
+    * Value name, units, absolute range, recommended range, default range
+    * VTE
+    * FiO2
+    * Humidity
+    * Temperature
+
+* Plots
+* Controlled Values
+
+    * PIP
+    * PEEP
+    * Inspiratory Time
+
+* Value Dependencies
+
+UI Notes & Todo===
+---------------
+
+* Jonny add notes from helpful RT in discord!!!
+* Top status Bar
+
+    * Start/stop button
+    * Status indicator - a clock that increments with heartbeats,
+        or some other visual indicator that things are alright
+    * Status bar - most recent alarm or notification w/ means of clearing
+    * Override to give 100% oxygen and silence all alarms
+
+* API
+
+    * Two queues, input and output. Read from socket and put directly into queue.
```

Comment:
We're thinking about revising this API to just getSensorValues() and setControlValues(). We can discuss more.